domain/src/main/java/me/androidbox/domain/DateTimeFormatterProvider.kt

```diff
@@ -12,4 +19,28 @@ object DateTimeFormatterProvider {
     fun ZonedDateTime.formatDateTime(pattern: String): String {
         return DateTimeFormatter.ofPattern(pattern, Locale.getDefault()).format(this)
     }
+
+    fun Long.toZoneDateTime(): ZonedDateTime {
+        val instant = Instant.ofEpochSecond(this)
+
+        return ZonedDateTime.ofInstant(instant, ZoneId.systemDefault())
+    }
+
+    fun LocalDate.toZoneDateTime(): ZonedDateTime {
+        return this.atStartOfDay(ZoneId.systemDefault())
+    }
+
+    fun AgendaItem.toDisplayDateTime(): String {
```

Comment:
Is this used somewhere inside the domain? Or just in presentation?
If only presentation, then that's where this method should be